src/components/FormField/FormMultiselect/FormMultiselect.tsx

```diff
@@ -0,0 +1,35 @@
+import Multiselect from "multiselect-react-dropdown";
+import {Label, ErrorSpan} from "../FromField.styled";
+import React from "react";
+
+export const FormMultiselect = (props: any) => {
+  const { field, form, ...rest } = props;
+  const error = form.errors[field.name];
+  const touched = form.touched[field.name];
+  return (
+    <>
+      <Label htmlFor={field.name}>{field.name}</Label>
+      <div
+        onBlur={() => {
+          form.setTouched({ ...form.touched, [field.name]: true });
+        }}
+      >
+        <Multiselect
+          {...field}
+          {...rest}
+          onRemove={(selectedList: any[]) => {
+            form.setFieldValue(field.name, selectedList);
+            form.setTouched({ ...form.touched, [field.name]: true });
```

Comment:
I think you don't need `setTouched` here anymore, `onBlur` will cover all the cases. Better test it though, I might be wrong.